### ISSUE TITLE
Should intrinsics be scoped?

### ISSUE THREAD

**Issue by linusg** [tc39/ecma402#842]
TL;DR: Is there anything speaking for, or against, renaming the intrinsics established in ECMA-402 to follow the form `%Intl.Foo%`? 

This may not be hugely important within the spec document itself, but personally I would appreciate if any references to those intrinsics in engine code can be more easily identified as `Intl`-related.

This may need input from the ECMA-262 editors - the closest relevant thing I could find is https://github.com/tc39/ecma262/commit/8f67a13e1af599fb31047c6f8e35acd7586d1bd1, which possibly sets a precedent for the names of intrinsics to match what they are referenced as from the outside world. The current Temporal draft spec uses `%Temporal.Foo%` for each constructor, opposed to a plain `%Foo%` name.

**Comment by ljharb** [comment]
It should be done for consistency with 262; I clearly forgot to follow through with that process after making the change in 262. HTML was already updated to follow the new convention.

**Comment by sffc** [comment]
@ben-allen @ryzokuken @gibson042

**Comment by gibson042** [comment]
I'm in favor, and commented on #843.
